jx_sqlite/expressions.py

```diff
@@ -61,11 +61,15 @@ def to_sql(self, schema, not_null=False, boolean=False):
                 tempb = tempa.setdefault(schema.get_column_name(col), {})
                 tempb[json_type_to_sql_type[col.type]] = quote_column(col.es_column).sql
 
-    return wrap([
-        {"name": relative_field(cname, self.var), "sql": types, "nested_path": nested_path}
-        for nested_path, pairs in acc.items() for cname, types in pairs.items()
-    ])
-
+    cols = []
+    for nested_path, pairs in acc.items():
+        for cname, types in pairs.items():
+            if not startswith_field(cname, self.var):
```

Comment:
I turned this into an Issue so we can deal with this later:

https://github.com/mozilla/jx-sqlite/issues/61